core/connection/interface.go

```diff
@@ -27,11 +27,18 @@ import (
 // DialogCreator creates new dialog between consumer and provider, using given contact information
 type DialogCreator func(consumerID, providerID identity.Identity, contact market.Contact) (communication.Dialog, error)
 
+// SessionCreationConfig are the parameters that get sent to the provider
+type SessionCreationConfig interface{}
```

Comment:
It's not about session creation config. It's about passing consumer config parameters to underlying transport - nothing to do with session itself